Apps/W1/EDocumentConnectors/SignUp/app/src/Connection.Codeunit.al

```diff
@@ -0,0 +1,105 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+namespace Microsoft.EServices.EDocumentConnector.SignUp;
+
+using Microsoft.EServices.EDocument;
+using Microsoft.Purchases.Document;
+using Microsoft.Purchases.Posting;
+using System.Utilities;
+
+codeunit 6382 Connection
+{
+    Access = Internal;
+    Permissions = tabledata "E-Document" = m;
+
+    procedure HandleSendFilePostRequest(var TempBlob: Codeunit "Temp Blob"; var EDocument: Record "E-Document"; var HttpRequest: HttpRequestMessage; var HttpResponse: HttpResponseMessage; Retry: Boolean): Boolean
```

Comment:
Is it recommended to do this retry thing or why are we doing it?

Reply:
Legacy based on template code... Removed